database/v2/index.js

```diff
@@ -0,0 +1,39 @@
+const models = require('./schemas');
+const Sequelize = require('sequelize');
+const { ErrorHandler } = require('../../api/middlewares/errorHandler');
+
+const db = {};
+
+const sequelize = new Sequelize(
+  process.env.DB_NAME,
+  process.env.DB_USER,
+  process.env.DB_PASSWORD,
+  {
+    host: process.env.DB_HOST,
+    dialect: 'postgres',
+    dialectOptions: {
+      ssl: true,
+    },
+    logging: true,
+  }
+);
+
+Object.keys(models).forEach(f => {
```

Comment:
What does 'f' mean?

Reply:
Ok, I will change this short name with a long readable one.